cfgov/prepaid_agreements/tests/test_scripts_import_prepaid_agreements.py

```diff
@@ -0,0 +1,138 @@
+                    "program_manager": "CFPB",
+                    "program_manager_exists": "Yes",
+                    "status": "Active",
+                    "withdrawal_date": "01/01/2020"
+                },
+                {
+                    "issuer_id": "2",
+                    "issuer_name": "CFPB Bank2",
+                    "other_relevant_parties": "Party",
+                    "prepaid_type": "Prison release",
+                    "product_id": "PRODUCT-2",
+                    "product_name": "Bank2_of_CFPB",
+                    "program_manager": "CFPB",
+                    "program_manager_exists": "Yes",
+                    "status": "Active",
+                    "withdrawal_date": "02/02/2020"
+                },
+                {
+                    "issuer_id": "3",
+                    "issuer_name": "CFPB Bank3",
+                    "other_relevant_parties": "Party",
+                    "prepaid_type": "Refunds",
+                    "product_id": "PRODUCT-3",
+                    "product_name": "Bank3 of CFPB",
+                    "program_manager": "CFPB",
+                    "program_manager_exists": "Yes",
+                    "status": "Active",
+                    "withdrawal_date": "03/03/2020"
+                },
+                {
+                    "issuer_id": "4",
+                    "issuer_name": "CFPB Bank4",
+                    "other_relevant_parties": "Party",
+                    "prepaid_type": "Student",
+                    "product_id": "PRODUCT-4",
+                    "product_name": "Bank4_of CFPB",
+                    "program_manager": "CFPB",
+                    "program_manager_exists": "Yes",
+                    "status": "Active",
+                    "withdrawal_date": "04/04/2020"
+                },
+                {
+                    "issuer_id": "5",
+                    "issuer_name": "CFPB Bank5",
+                    "other_relevant_parties": "Party",
+                    "prepaid_type": "Tax",
+                    "product_id": "PRODUCT-5",
+                    "product_name": "Bank5 of CFPB",
+                    "program_manager": "CFPB",
+                    "program_manager_exists": "Yes",
+                    "status": "Active",
+                    "withdrawal_date": "05/05/2020"
+                },
+                {
+                    "issuer_id": "6",
+                    "issuer_name": "CFPB Bank6",
+                    "other_relevant_parties": "Party",
+                    "prepaid_type": "Travel",
+                    "product_id": "PRODUCT-6",
+                    "product_name": "Bank6_of_CFPB",
+                    "program_manager": "CFPB",
+                    "program_manager_exists": "Yes",
+                    "status": "Active",
+                    "withdrawal_date": "06/06/2020"
+                }
+            ]
+        }
+        import_products_data(data['products'])
+
+        import_agreements_data(data['agreements'])
```

Comment:
Do we want to make any assertions about the success of the imports?

Reply:
Great suggestion. I added assertEqual to check return value from imports.